lib/ssl/doc/src/ssl.xml

```diff
@@ -1748,11 +1748,15 @@ fun(srp, Username :: binary(), UserState :: term()) ->
     
     <func>
       <name name="eccs" arity="0" since="OTP 19.2"/>
+      <fsummary>Returns a list of supported elliptic curves.</fsummary>
+      <desc><p>Returns a list of supported elliptic curves pre TLS-1.3.</p>
+      </desc>
+    </func>
+
+    <func>
       <name name="eccs" arity="1" since="OTP 19.2"/>
-      <fsummary>Returns a list of supported ECCs.</fsummary>
-      <desc><p>Returns a list of supported ECCs. <c>eccs()</c>
-      is equivalent to calling <c>eccs(Protocol)</c> with all
-      supported protocols and then deduplicating the output.</p>
+      <fsummary>Returns a list of default supported elliptic curves.</fsummary>
+      <desc><p>Returns a list of default supported elliptic curves for <c>Version</c> </p>
```

Comment:
could be my poor english, but distinction between:_a list of default supported_ and _a list of supported_ is very small. maybe `default supported` could/should be extended to something more obvious - so there is less space for speculation and being picky ;-)

maybe `supported` is obvious for something that is `default`?

Reply:
Refreshed, please read again.